LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-invalid.tentative.html

```diff
@@ -30,6 +30,7 @@
 test_invalid_number('progress(0,,, 10,, 200)');
 test_invalid_number('progress(0,,, 10,,, 200)');
 test_invalid_number('progress(0,, 10,, 200)');
+test_invalid_number('progress(0, 10, 200)');
```

Comment:
Oh man, I just remove this case, but I guess haven't upstreamed that one yet.

Reply:
I exported this here: https://github.com/web-platform-tests/wpt/commit/0801098bacff2ebf3a304c3cf4400b6fcbe2697a